rails/app/assets/stylesheets/components/welcome.scss

```diff
@@ -8,6 +8,18 @@
   align-items: center;
   text-align: center;
   
+  .logo {
+  
+		 justify-content: center;
+		align-items: center;
+		text-align: center;
+	
+		img {
+			height: 50%;
```

Comment:
Probably don't need a height here; setting the width to the desired size will ensure the logo remains the proper ratio.

Reply:
Fixed in updated style sheet. Thanks!